### PR TITLE
Add Ripple

### PR DESCRIPTION
**Pull request by vhpoet**
Awesome list of Ripple (open-source distributed settlement network) resources

### CODE REVIEW

**Comment by sindresorhus** [comment]
Please read the [contributing guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [awesome manifest](https://github.com/sindresorhus/awesome/blob/master/awesome.md).

Especially this part:

> Link additions should be added to the bottom of the relevant category.

**Comment by sindresorhus** [comment]
Could use some descriptions on https://github.com/vhpoet/awesome-ripple/#gateways--bridges

I would also add a short description at the top of your readme on what Ripple is.

File: readme.md [hunk 1]
```diff
@@ -304,6 +304,7 @@ Please read the [contribution guidelines](contributing.md) or the [creating a li
 - [Services Engineering](https://github.com/mmcgrana/services-engineering)
 - [Free for Developers](https://github.com/ripienaar/free-for-dev)
 - [Bitcoin](https://github.com/igorbarinov/awesome-bitcoin/)
+- [Ripple](https://github.com/vhpoet/awesome-ripple/)
```
Comment (sindresorhus) [inline readme.md]: Can you add a tiny description (like the below) on what Ripple is?

**Comment by vhpoet** [comment]
> Link additions should be added to the bottom of the relevant category.

Sorry for that, they're somewhat related so I thought it might make more sense to put it there. Fixed. Maybe we should actually create a new category for payments?

> Can you add a tiny description (like the below) on what Ripple is?

Done

**Comment by sindresorhus** [comment]
I don't think it's worth having a category for payments with only two items. Maybe if the list gets another one.

**Pull request closed without merging**